website/content/docs/secrets/azure.mdx

```diff
@@ -56,28 +88,60 @@ management tool.
 
    Success! Data written to: azure/config
    ```
+</Tab>
+
+<Tab heading="Vault UI">
 
-   If you are running Vault inside an Azure VM with MSI enabled, `client_id` and
-   `client_secret` may be omitted. For more information on authentication, see the [authentication](#authentication) section below.
+1. Click **Configure** from the Azure secret engine configuration view.
+![Partial screenshot of the Vault GUI showing the "Configure" Toolbar option](/img/gui/azure/configure-prompt.png)
+1. On the configuration form enter your configuration information. If you are an Enterprise user, you will see the option to select Access Type.
+![Partial screenshot of the Vault GUI showing the Configuration form](/img/gui/azure/configure.png)
+1. Save your changes.
 
-   In some cases, you cannot set sensitive account credentials in your
-   Vault configuration. For example, your organization may require that all
-   security credentials are short-lived or explicitly tied to a machine identity.
+</Tab>
 
-   To provide managed identity security credentials to Vault, we recommend using Vault
-   [plugin workload identity federation](#plugin-workload-identity-federation-wif)
-   (WIF) as shown below.
+</Tabs>
 
-1.  Alternatively, configure the audience claim value and the Client, Tenant and Subscription IDs for plugin workload identity federation:
+If you are running Vault inside an Azure VM with MSI enabled, `client_id` and `client_secret` may be omitted. For more information on authentication, see the [authentication](#authentication) section below.
 
-   ```text
+In some cases, you cannot set sensitive account credentials in your Vault configuration. For example, your organization may require that all security credentials are short-lived or explicitly tied to a machine identity.
+
+To provide managed identity security credentials to Vault, we recommend using Vault
+[plugin workload identity federation](#plugin-workload-identity-federation-wif)
+(WIF) as shown below.
+
+<Note> Workload identity federation is only available with Vault Enterprise. </Note>
```

Comment:
```suggestion
```
We should use the "enterprise-only" or "enterprise-and-hcp" partials at the top of the relevant section/page rather than adding an adhoc alert here.